src/content/docs/apm/agents/python-agent/python-agent-api/getbrowsertimingheader-python-agent-api.mdx

```diff
@@ -17,19 +17,55 @@ freshnessValidatedDate: never
 ## Syntax
 
 ```py
-get_browser_timing_header()
+get_browser_timing_header(nonce=None)
 ```
 
 Generates a browser agent header for injecting the JavaScript agent.
 
 ## Description
 
-This call returns the browser agent JavaScript header snippet that you can use to enable [browser monitoring](/docs/agents/python-agent/supported-features/page-load-timing-python). You might use this if automatic instrumentation is not possible for your application, or if you want to collect browser data for only specific pages.
+<Callout variant="important">
+The nonce parameter for nonced CSP support was added in Python Agent version 9.4.0+.
+
+In versions 9.3.0 and below, the [`get_browser_timing_footer`](/docs/agents/python-agent/python-agent-api/get_browser_timing_footer) API was also required to enable browser timing. Starting with version 9.4.0+ this is no longer required.
```

Comment:
```suggestion
In Python agent versions 9.3.0 and below, the [`get_browser_timing_footer`](/docs/agents/python-agent/python-agent-api/get_browser_timing_footer) API was also required to enable browser timing. Starting with version agent 9.4.0 and later, this is no longer required.
```

Reply:
This one should probably be this way around? agent version not version agent?
```suggestion
In Python agent versions 9.3.0 and below, the [`get_browser_timing_footer`](/docs/agents/python-agent/python-agent-api/get_browser_timing_footer) API was also required to enable browser timing. Starting with agent version 9.4.0 and later, this is no longer required.
```

Reply:
Yes, good catch!